DI-435/merge_VCF_AF.sh

```
@@ -12,6 +12,33 @@ genome=$2
+PS4='\000[$(date)]\011'
+export TZ=Europe/London
+
+# set frequency of instance usage in logs to 10 seconds
+sudo kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
+/usr/bin/dx-dstat 10
+
+_get_peak_usage() {
+    : '''
+    Reports the peak memory and storage usage from dstat, to be called at end of script
+    '''
+    dx watch $DX_JOB_ID --no-follow --quiet > job.log
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)